chromium_src/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.h

```diff
@@ -46,4 +50,4 @@ using TabDragControllerBrave = TabDragController;
 
 #include "brave/browser/ui/views/tabs/tab_drag_controller.h"
```

Comment:
Just wondering if we should do a similar move for our implementation.

Reply:
Makes sense, fixed here https://github.com/brave/brave-core/pull/27172/commits/d0678d87f8bb737d5e19f7f9322f22e004775fc5.